controller/users_blackbox_test.go

```diff
@@ -804,6 +804,53 @@ func (s *TestUsersSuite) TestListUsersByEmailOKEmptyResult() {
 	require.Len(s.T(), result.Data, 0)
 }
 
+func (s *TestUsersSuite) TestHideEmailOK() {
+	boolTrue := true
+
+	// given user1
+	user1 := s.createRandomUser("TestListUsersOK1")
+	identity := s.createRandomIdentity(user1, account.KeycloakIDP)
+
+	// when
+	email := user1.Email
+
+	// by default, email is public.
+	_, result := test.ListUsersOK(s.T(), nil, nil, s.controller, &email, nil, nil, nil)
```

Comment:
Also check if test.ShowUsersOK() returns the public email.

Reply:
6583a918d37a389ab2aba36e546e01c89965f419